### PR TITLE
Update examples to work with global stats instance

### PR DESCRIPTION
**Pull request by mayurkale22**
This PR contains updated examples based on CHAGELOG -> https://github.com/census-instrumentation/opencensus-node/blob/master/CHANGELOG.md#new-code

Also, added ReadMe for the Stackdriver Stats(Metrics) exporter.

### CODE REVIEW

**Comment by mayurkale22** [comment]
+ @DazWilkin for review.

File: packages/opencensus-exporter-stackdriver/README.md [hunk 1]
```diff
@@ -55,6 +56,58 @@ or
...
+
+## Usage
+
+To use Stackdriver as your exporter, make sure you have enabled [Stackdriver Monitoring](https://cloud.google.com/monitoring/docs/quickstart) on Google Cloud Platform. Enable your [Application Default Credentials](https://cloud.google.com/docs/authentication/getting-started) for authentication with:
+
+```bash
+export GOOGLE_APPLICATION_CREDENTIALS=path/to/your/credential.json
+```
+
+Instance the exporter on your application.
```
Comment (draffensperger) [inline packages/opencensus-exporter-stackdriver/README.md]: Nit: did you mean "Install the exporter ..."?

Comment (mayurkale22) [reply packages/opencensus-exporter-stackdriver/README.md]: Copied from top example (Trace Exporter), amended both the sentence.

**Comment by mayurkale22** [comment]
+@alexamies for review (especially ```web_client_monitoring``` example)

**Comment by alexamies** [comment]
> +@alexamies for review (especially `web_client_monitoring` example)

Looks good. Thanks for making the changes in the web_client_monitoring example.

File: packages/opencensus-exporter-stackdriver/README.md (hunk 1 above)
Comment (DazWilkin) [reply packages/opencensus-exporter-stackdriver/README.md]: Yes, I did.

**Pull request merged**